app/src/main/res/resource/values-de/strings.xml

```diff
@@ -154,30 +156,30 @@
     <string name="feedback_exceed_rate_limit">Ups! Es gab zu viele Besucher in dieser Stunde, bitte versuchen Sie in der nächsten Stunde erneut.</string>
 
     <string name="feedback_confirm_change_theme">Themaänderung bestätigen</string>
-    <string name="feedback_change_theme_warning">Jetztiges Ändern des Themas wird aktuelle downloads abbrechen.</string>
+    <string name="feedback_change_theme_warning">Änderung des Themas wird alle aktuelle Downloads abbrechen.</string>
     <string name="feedback_confirm_restart">Neustart bestätigen</string>
-    <string name="feedback_restart_warning">Jetztiges Neustarten der App wird alle Downloads abbrechen.</string>
+    <string name="feedback_restart_warning">Neustarten der App wird alle aktuellen Downloads abbrechen.</string>
 
     <string name="feedback_click_again_to_exit">Zum Verlassen nochmal drücken.</string>
 
     <string name="feedback_share_photo_title">Ein Foto von Unsplash.com</string>
-    <string name="feedback_share_photo_extra">Genommen von # am #</string>
+    <string name="feedback_share_photo_extra">Genommen von # am #\n</string>
     <string name="feedback_share_collection_title">Eine Kollektion von Unsplash.com</string>
-    <string name="feedback_share_collection_extra">Erstellt von # am #</string>
+    <string name="feedback_share_collection_extra">Erstellt von # am #\n</string>
     <string name="feedback_share_user_title">Ein Benutzerprofil von Unsplash.com</string>
-    <string name="feedback_share_user_extra">#</string>
+    <string name="feedback_share_user_extra">#\n</string>
 
-    <string name="feedback_like_failed">Liken des Fotos fehlgeschlagen.</string>
-    <string name="feedback_unlike_failed">Unliken des Fotos fehlgeschlagen.</string>
+    <string name="feedback_like_failed">Favorisieren des Fotos fehlgeschlagen.</string>
+    <string name="feedback_unlike_failed">Als Favorit entfernen fehlgeschlagen.</string>
```

Comment:
Sounds strange, but difficult to translate. Maybe "Favorisierung entfernen", "Favorit löschen", "Entfernen des Favoriten[statuses]"…